frontend/lib/pages/hp-action-access-for-inspection.tsx

```diff
@@ -5,14 +5,21 @@ import { TextualFormField } from '../form-fields';
 
 const onboardingStepBuilder = new SessionStepBuilder(sess => sess.onboardingInfo);
 
-export const AccessForInspection = onboardingStepBuilder.createStep({
+type HpType = 'hp' | 'ehp';
+
+const AccessForInspectionGenerator = (type?: HpType) => onboardingStepBuilder.createStep({
   title: "Access for Your HPD Inspection",
   mutation: AccessForInspectionMutation,
   toFormInput: onb => onb.finish(),
   renderIntro: () => <>
-    <p>On the day of your HPD Inspection, the Inspector will need access to your apartment during a window of time that you will choose with the HP Clerk when you submit your paperwork in Court.</p>
+    <p>On the day of your HPD Inspection, the Inspector will need access to your apartment during a window of time that you will choose with 
+      {' '}{type === 'ehp' ? "your lawyer" : "the HP Clerk when you submit your paperwork in Court"}.
+    </p>
   </>,
   renderForm: ctx => <>
     <TextualFormField {...ctx.fieldPropsFor('floorNumber')} type="number" min="0" label="What floor do you live on?" />
   </>,
 });
+
+export const AccessForInspection = AccessForInspectionGenerator('hp');
```

Comment:
Since hp is the default version to render, we don't actually need this `hp` argument here but thought i'd add for clarity and readability. Is this best practice?

Reply:
cool cool, good to know

Reply:
er to clarify i guess it is really a matter of taste/interpretation rather than best practice... one could argue that a "normal" hp action should be the default while an EHPA should be the special case but i thought it made more sense to list out the options and force one to be passed in explicitly... but i guess both of these are valid!

another way to do it might have been to have an `isEmergencyHPA` boolean argument but i don't like that b/c just passing in a boolean as an argument can be confusing to read!

Reply:
Yeah definitely! I originally implemented a Boolean and had the same
reaction to it. I like this approach bc now we can add all different types
of HP action flows and scale easily!